Ghidra/Features/Decompiler/ghidra_scripts/ShowConstantUse.java

```diff
@@ -867,12 +867,12 @@ private void followToParam(HashMap<Address, Long> constUse, ArrayList<PcodeOp> d
 					doneSet);
 				return;
 			case PcodeOp.MULTIEQUAL:
-				followToParam(constUse, defUseList, highFunction, def.getInput(0), funcList,
-					doneSet);
-				@SuppressWarnings("unchecked")
-				ArrayList<PcodeOp> splitUseList = (ArrayList<PcodeOp>) defUseList.clone();
-				followToParam(constUse, splitUseList, highFunction, def.getInput(1), funcList,
-					doneSet);
+				for (int i = 0; i < def.getNumInputs(); i++) {
+					@SuppressWarnings("unchecked")
+					ArrayList<PcodeOp> splitUseList = (ArrayList<PcodeOp>) defUseList.clone();
```

Comment:
The ```defUseList.clone()``` seems like its some crusty code that shouldn't be copied or used as a template for well-formed code.

I'm pretty sure it should be replaced with something like: ```ArrayList<PcodeOp> splitUseList = new ArrayList<>( defUseList );```

You will be able to get rid of the ```@SuppressWarning``` was well.

Reply:
removed annotation & clone()